### DISCUSSION TITLE
ToDoアプリを作っています

### DISCUSSION THREAD

**Discussion by KOTARO1209** [lit-iphone-course/lit-iphone-course#143] [Question-original]
## (1) 実現したいこと
スワイプ削除は最後に回して、まずはアプリを実行できるようにしたい。（立ち上げて、名前金額を入力してリストに反映させたい）
## (2) 実現するために自分が試した内容とその結果
前回の質問からスワイプ削除がずっとできなかったため、プロジェクトを作り直してRealmの教科書をもう一度やりました。一通り完成させてからカテゴリー部分を消しました。（ToDoアプリには不要だったので）
その後実行しようとしたらnilに関するエラーが出ました。調べたところnilやアンラップに関するエラーであることは分かったのですが、そこからの解決方法がわからないのでお聞きしたいです。
## (3) 調べた内容や資料
教科書Realmの1と2
https://qiita.com/nsy_python/items/372e6e3737c67bada9f5
## (4) 原因だと思われそうな箇所(ソースコードをコピペして貼る)
![image](https://user-images.githubusercontent.com/75009903/220588780-6bee3f5f-a474-4d58-addc-63e45f10c551.png)
## (5) GitHubのリンク
https://github.com/KOTARO1209/ToDoList

**Comment by txxxxc** [comment]
あ、ごめん！こっちのカテゴリーで質問投稿してもらっていい？？
https://github.com/lit-iphone-course/lit-iphone-course/discussions/new?category=question-original
